zulipterminal/model.py

```diff
@@ -557,8 +557,8 @@ def send_stream_message(self, stream: str, topic: str, content: str) -> bool:
             notify_if_message_sent_outside_narrow(composition, self.controller)
         return message_was_sent
 
-    def get_file_upld_uri(self, file_loctn: str) -> str:
-        with open(file_loctn, "rb") as fp:
+    def get_file_upld_uri(self, file_loc: str) -> str:
+        with open(file_loc, "rb") as fp:
```

Comment:
I think this can be added to the second commit, where the function has been declared?